docs/macros.rst

```diff
@@ -510,6 +512,10 @@ API
     :param header_classes: A string of classes to apply to the table header (e.g ``'thead-dark'``).
     :param responsive: Whether to enable/disable table responsiveness.
     :param responsive_class: The responsive class to apply to the table. Default is ``'table-responsive'``.
+    :param safe: Tuple with columns names to render HTML safe using ``|safe``.
```

Comment:
Maybe using `safe_columns` and `urlize_columns` will be better.

Reply:
Refactored